shoop/guide/settings.py

```diff
@@ -0,0 +1,15 @@
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+SHOOP_GUIDE_API_URL = "https://readthedocs.org/api/v2/search/?project=shoop-guide&version=latest&"
+SHOOP_GUIDE_LINK_URL = "http://shoop-guide.readthedocs.io/en/latest/search.html?check_keywords=yes&area=default&"
+
+# If true, fetch results via the ReadtheDocs API, otherwise
+# only display a link to RTD search page
+SHOOP_GUIDE_FETCH_RESULTS = True
+
+# Timeout limit in seconds for fetching search results
+SHOOP_GUIDE_TIMEOUT_LIMIT = 2
```

Comment:
Document these settings with correct `#:` formatting so that Sphinx and `shoop_show_settings` management command can parse it. There should be title line and body for each setting. See e.g. the comments of SHOOP_PRICING_MODULE or SHOOP_DISCOUNT_MODULES in shoop/core/settings.py for example.

(This is how they appear on docs btw: http://shoop.readthedocs.io/en/latest/api/shoop.core.html#shoop.core.settings.SHOOP_PRICING_MODULE)